### PR TITLE
[WIP] Create Episode Proposal Template

### PR DESCRIPTION
**Pull request by garricn**
In conjunction with #18 an Episode Proposal Template could be useful to make it easy for people to propose new episode ideas. This is inspired by both the Swift Weekly Brief process for contributing issues and the Swift evolution process for proposing language changes.

### CODE REVIEW

**Comment by BasThomas** [comment]
Would we want to make this an [issue template](https://help.github.com/articles/creating-issue-templates-for-your-repository/) so the process of using the template is a bit easier? This sounds perfect for it!

File: Episode-Proposal-Template [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Episode Proposal Template
+
+# [TITLE]
+
+* Proposal: [PR#n]([link to PR])
```
Comment (BasThomas) [inline Episode-Proposal-Template]: Would this need an associated pull request, or will it just be an issue?

Comment (BasThomas) [reply Episode-Proposal-Template]: Or are you thinking to have all proposals be a pull request, like with swift-evolution?

File: Episode-Proposal-Template [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Authors: [Name]([github profile], ...
+* Review Manager: [Name]([github profile])
```
[the remark this answers is not in the document]
Comment (BasThomas) [reply Episode-Proposal-Template]: I like this, especially to have one contact person for questions and to keep the status up-to-date.

File: Episode-Proposal-Template [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Motivation
+
+Brief description of your motivation behind this episode
+
+## Proposed contribution
+
+Brief description of how you intend to contribute to this episode and help make it a reality
+
+## Source compatibility
```
Comment (BasThomas) [inline Episode-Proposal-Template]: 😄

File: Episode-Proposal-Template (hunk 1 above)
Comment (garricn) [reply Episode-Proposal-Template]: I think all proposals would be a PR and that the proposal would be updated to include the PR# and link to PR. Might be overkill but I will try and see. what do you think?

File: Episode-Proposal-Template [hunk 4]
```diff
@@ -0,0 +1,37 @@
+Episode Proposal Template
+
+# [TITLE]
+
+* Proposal: [PR#n]([link to PR])
+* Authors: [Name]([github profile], ...
+* Review Manager: [Name]([github profile])
+* Status: [Proposed, Pre-production, Production, Post-production, Published](link to episode if published)
```
Comment (garricn) [inline Episode-Proposal-Template]: status would be updated overtime just like swift evolution. could be tedious but could be fun :)

File: Episode-Proposal-Template (hunk 1 above)
Comment (BasThomas) [reply Episode-Proposal-Template]: I think that sounds good!

**Comment by garricn** [comment]
> Would we want to make this an [issue template](https://help.github.com/articles/creating-issue-templates-for-your-repository/) so the process of using the template is a bit easier? This sounds perfect for it!

Do you think an Issue as opposed to a PR would serve the Episode Proposal concept better? Or am I misunderstanding your suggestion...?

**Comment by applecoding** [comment]
I think will be good a field of "target audience". It's not the same people that approaching the podcast with advanced profiles and higher level topics such as library implementations or people who come to the podcast out of curiosity and to learn, who want to enter the world of Swift. I think there should be room for all of them. For example: basic, intermediate, advanced and expert as values in target audience. In that way, people will know exactly what they're going to hear.

**Comment by BasThomas** [comment]
> Do you think an Issue as opposed to a PR would serve the Episode Proposal concept better? Or am I misunderstanding your suggestion...?

After thinking about it a bit more, I think I was on the wrong path — doing this through actual pull requests sounds a lot better. So disregard what I suggested there :)

**Comment by garricn** [comment]
> I think will be good a field of "target audience". It's not the same people that approaching the podcast with advanced profiles and higher level topics such as library implementations or people who come to the podcast out of curiosity and to learn, who want to enter the world of Swift. I think there should be room for all of them. For example: basic, intermediate, advanced and expert as values in target audience. In that way, people will know exactly what they're going to hear.

@applecoding I think its a good idea. I am going to merge this now and then feel free to open a PR that adds that.

**Pull request merged**